geth/jail/jail.go

```diff
@@ -128,47 +131,54 @@ func (j *Jail) initCell(cell *Cell) error {
 }
 
 // CreateAndInitCell creates and initializes a new Cell.
-func (j *Jail) createAndInitCell(chatID string, code ...string) (*Cell, error) {
+func (j *Jail) createAndInitCell(chatID string, code ...string) (*Cell, string, error) {
```

Comment:
I'm a bit hesitant about returning 3 values here, but it seems like the simplest solution.
We use both `Cell` and `result` in the callers'.